temporalcli/commandsgen/commands.yml

```diff
@@ -3035,6 +3000,7 @@ option-sets:
         env: TEMPORAL_CODEC_AUTH
 
   - name: overlap-policy
+    description: Schedule commands
```

Comment:
The option-set `description` is a new field added for `option-set` usages in the [the combined approach](https://github.com/prasek/temporal-cli/tree/cli-docs-gen-all), where it would show up as follows:

<img width="574" alt="image" src="https://github.com/user-attachments/assets/9eef4ff2-a595-4e46-8ec9-28bd10d34376">

Until the [the combined approach](https://github.com/prasek/temporal-cli/tree/cli-docs-gen-all) lands this field is not used.

Note the main purpose is to to see where divergent option descriptions are used (in commands or option-sets) in the generated `cmd-options.mdx` that collate on the various option descriptions under a given option name across command and option set usages.

After discussion with @yuandrew it's likely these would just be used for linting rules as we're moving away from `cmd-options.mdx` as it doesn't scale very well.

Given that will remove all these option-set descriptions for now and we can revisit later.

Reply:
Ah, that's what you were going for. Sorry I was so confused. Thanks for the visual aid.